stdlib/2and3/argparse.pyi

```diff
@@ -283,6 +284,21 @@ class Action(_AttributeHolder):
     if sys.version_info >= (3, 9):
         def format_usage(self) -> str: ...
 
+if sys.version_info >= (3, 9):
+    class BooleanOptionalAction(Action):
+        def __init__(
+            self,
+            option_strings: Sequence[Text],
+            dest: Text,
+            const: Any = ...,
```

Comment:
I don't fully understand the description of this argument but it sounds like it and default should be of type `Optional[_T]`.